models/articles.model.js

```diff
@@ -1,14 +1,27 @@
 const db = require("../db/connection")
 
-exports.selectArticleById = (article_id) =>{
+exports.selectArticleById = (article_id, comment_count) =>{
+
+  if(!comment_count){
```

Comment:
We won't need any of this conditional logic as per my feedback in your tests, this comment_count property should always be on this response by default.